app/app/(app-active-cadets)/active-cadets/layout.tsx

```diff
@@ -0,0 +1,16 @@
+export const metadata = {
+  title: 'Next.js',
+  description: 'Generated by Next.js',
+}
+
+export default function RootLayout({
```

Comment:
Add <app name> to the rootlayout name
 
Easy debugging when things are specific

e.g `ActiveCadetsRootLayout`